zebra-chain/src/parameters/network_upgrade.rs

```diff
@@ -301,6 +301,15 @@ impl NetworkUpgrade {
             Network::Testnet => height >= TESTNET_MAX_TIME_START_HEIGHT,
         }
     }
+    /// Returns the NetworkUpgrade given an u32 as ConsensusBranchId
+    pub fn network_upgrade(branch_id: u32) -> Option<NetworkUpgrade> {
```

Comment:
This function name is a bit confusing, we'll write code like `NetworkUpgrade::network_upgrade(branch_id)`.

In Rust we usually say "from…":
```suggestion
    pub fn from_branch_id(branch_id: u32) -> Option<NetworkUpgrade> {
```